### ISSUE TITLE
Support uploading files with non-latin characters in filenames

### ISSUE THREAD

**Issue by fishfree** [fcrepo/fcrepo#1958]
When uploading a file with non-latin characers in the filename, the item URL get a 404 error.

**Comment by pwinckles** [comment]
What version of Fedora are you using?

**Comment by fishfree** [comment]
@pwinckles  I use this repo https://github.com/fcrepo-exts/fcrepo-docker, so I think it's the latest version?

**Comment by pwinckles** [comment]
This should have been fixed by https://github.com/fcrepo/fcrepo/pull/1945 which was released in the 6.1.0 release.

For example:

```shell
docker run -it --rm -p 8080:8080 docker.io/fcrepo/fcrepo:6.1.1

curl -v -H'Link: <http://www.w3.org/ns/ldp#NonRDFSource>; rel="type"' -H"Content-Disposition: attachment; filename*=UTF-8''%E6%B5%85%E9%A1%B6%E8%BD%AF%E5%91%A2%E5%B8%BD%E5%B2%A9%E7%9F%B3" -H"Content-type: text/plain" -XPUT http://localhost:8080/fcrepo/rest/bin -ufedoraAdmin:fedoraAdmin --data-binary "testing"

curl http://localhost:8080/fcrepo/rest/bin -ufedoraAdmin:fedoraAdmin
```

Note that the filename must be specified like `filename*=UTF-8''URL_ENCODED_NAME`.

Let me know if you're able to get it to work.

**Comment by whikloj** [comment]
@fishfree were you able to resolve this issue?

**Comment by Surfrdan** [comment]
Closing due to a lack of activity. If this was not resolved properly with 6.1 please do re-open
